Compiler/General.py

```diff
@@ -235,14 +235,18 @@ def get_token_code(ids_map_list, token, current_pointer):
     raise NotImplementedError
 
 
-def get_divmod_code():
+def get_divmod_code(right_token=None):
     # given that the current pointer points to a, and the cell after a contains b,
     # (i.e the cells look like: --> a, b, ?, ?, ?, ?, ...)
     # returns a code that calculates divmod, and the cells look like this:
     # --> 0, b-a%b, a%b, a/b, 0, 0
     # and the pointer points to the first 0 (which is in the same cell as a used to be)
     ADD_DIVISION_BY_ZERO_CHECK = True
 
+    if right_token is not None and right_token.type == Token.NUM:
+        if right_token.data != 0:
```

Comment:
For Token.NUM, the "data" property contains string, so this will actually omit the division by zero even when NUM is zero. Please use get_NUM_token_value

Reply:
Thanks for reminding that the num token doesn't have a int